### PR TITLE
Adding missing elements to root pom.xml

### PR DESCRIPTION
**Pull request by danbev**
Motivation:
The root pom is missing the following elements:
- licences
- developers
- scm
- profile for release-sign-artifacts

Modifications:
Added the missing sections.

Result:
Should be able to run the release process now.

### CODE REVIEW

**Comment by danbev** [comment]
@abstractj @danielpassos Would you guys mind to review this PR?

File: pom.xml [hunk 1]
```diff
@@ -34,6 +34,29 @@
         <version>0.2.3</version>
```
Comment (matzew) [inline pom.xml]: let's get the version updated to use the latest

File: pom.xml [hunk 2]
```diff
@@ -34,6 +34,29 @@
...
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <id>aerogear</id>
+            <name>AeroGear Team</name>
+            <email>aerogear-dev@lists.jboss.org</email>
+        </developer>
+    </developers>
```
Comment (matzew) [inline pom.xml]: License and developers are inherited from the parent, no need to duplicate it here

File: pom.xml [hunk 3]
```diff
@@ -283,6 +306,36 @@
...
+                                <goals>
+                                    <goal>sign</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
```
Comment (matzew) [inline pom.xml]: this profile is also available at the parent

File: pom.xml (hunk 1 above)
Comment (danbev) [inline pom.xml]: Fixed

File: pom.xml (hunk 2 above)
Comment (danbev) [inline pom.xml]: Done

File: pom.xml (hunk 3 above)
Comment (danbev) [inline pom.xml]: Done

**Comment by matzew** [comment]
:+1:

**Comment by danbev** [comment]
@matzew Thanks!
Merged as fb727cf

**Comment by danbev** [comment]
@danielpassos @abstractj Sorry about merging before you had a chance to review. Hope you don't mind.

**Pull request closed without merging**